frontend/src/app/products/page.tsx

```
@@ -0,0 +1,97 @@
+              initial={{ x: '-100%' }}
+              animate={{ x: 0 }}
+              exit={{ x: '-100%' }}
+              transition={{ type: 'spring', stiffness: 200, damping: 24 }}
+            >
+              <div className="flex justify-between items-center mb-6">
+                <h2 className="text-2xl font-medium text-black">Filtros</h2>
+                
+                <button
+                  onClick={() => setShowFilters(false)}
+                  className="text-sm text-gray-500 hover:text-black transition"
+                >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)